build.gradle

```diff
@@ -2596,6 +2626,7 @@ project(':streams') {
     implementation libs.slf4jApi
     implementation libs.jacksonAnnotations
     implementation libs.jacksonDatabind
+    implementation libs.bndlib
```

Comment:
Hi @mumrah 
I add this to fix the warning during build:
```
/home/frankvicky/.gradle/caches/modules-2/files-2.1/org.apache.logging.log4j/log4j-api/2.24.1/7ebeb12c20606373005af4232cd0ecca72613dda/log4j-api-2.24.1.jar(/org/apache/logging/log4j/Level.class): warning: Cannot find annotation method 'value()' in type 'BaselineIgnore': class file for aQute.bnd.annotation.baseline.BaselineIgnore not found
```

Reply:
Fair enough, I will try to solve this one.

Reply:
The BND annotations are intentionally in the `provided` Maven scope of all Log4j artifacts, so that these annotations with `CLASS` retention do not end up in the runtime classpath. You can do the same and add them as `compileOnly` in Gradle.

The compiler warnings should disappear once [JDK-8342833](https://bugs.openjdk.org/browse/JDK-8342833) is fixed.
Untile then we will remove the outdated ones (see apache/logging-log4j2#3133) in the next Log4j release, which should remove the warning on `Level`.

Reply:
Thanks for the information.  
I have already changed its scope to compile time.
PTAL 😄